webapp/src/components/sidebar_right/github_items.tsx

```diff
@@ -381,11 +443,18 @@ function getReviewText(item: Item, style: any, secondLine: boolean) {
     }
 
     if (changesRequested > 0) {
+        // there are two conflicting rules so decided to choose one over the other on line 456
```

Comment:
Please remove this comment. Plus the line numbers will change over time, so this will eventually be referencing the wrong line. Also, this should probably be a PR comment, not a comment in the code.

Reply:
forgot to delete this one.